src/stylesheets/components/_auth-alert.scss

```diff
@@ -1,28 +1,44 @@
 .auth-alert {
-  backdrop-filter: blur(rem(54px));
-  background: map-get($brand-gray, 'primary');
-  border-radius: rem(12px);
-  margin-bottom: rem(24px);
+  @extend .row;
```

Comment:
💡 Just a reminder that components should not be concerned by container/rows as they should work in any layouts. Think of styling them in isolation (on a related topic, using [Storybook](https://storybook.js.org/docs/react/get-started/introduction) is great to document and illustrate this idea). In other words, build them at 100% screen width.

Then use screen stylesheets to override or add the correct styles to make it work on a specific page (as done in [_login.scss](https://github.com/rafayet-monon/surveyor/pull/13/files#diff-26c3626dda3a7c4dbd2c6de1c4e49f92)).

Reply:
Updated in [ffd443a](https://github.com/rafayet-monon/surveyor/pull/13/commits/ffd443aef2909ad5840697b957e49a457f1eaf94). Will begin to use [Storybook](https://storybook.js.org/docs/react/get-started/introduction) along the way 🚀